### PR TITLE
PAYARA-2245 integrated Mojara 2.4 and updated SPI impl for Mojarra 2.4

### PR DESCRIPTION
**Pull request by arjantijms**
Payara-2245 large xml file parsing addressed

### CODE REVIEW

**Comment by arjantijms** [comment]
jenkins quick test payara 5

**Comment by arjantijms** [comment]
jenkins quick test payara 5

**Comment by arjantijms** [comment]
jenkins quick test payara 5

File: nucleus/common/glassfish-api/src/main/java/org/glassfish/api/invocation/InvocationManager.java [hunk 1]
```diff
@@ -37,8 +37,7 @@
  * only if the new code is made subject to such option by the copyright
  * holder.
  */
-// Portions Copyright [2016] [Payara Foundation]
-
+// Portions Copyright [2017] [Payara Foundation and/or its affiliates]
```
Comment (michaelranaldo) [inline nucleus/common/glassfish-api/src/main/java/org/glassfish/api/invocation/InvocationManager.java]: should this be 2016-2017?

Comment (arjantijms) [reply nucleus/common/glassfish-api/src/main/java/org/glassfish/api/invocation/InvocationManager.java]: Good catch, I think so indeed

**Comment by arjantijms** [comment]
jenkins quick test payara 5

**Comment by arjantijms** [comment]
jenkins quick test payara 5

**Pull request merged**